lib/dashboard-template.hbs

```diff
@@ -633,10 +633,9 @@ input:checked + .slider:before {
             <button id="topOfRequestsScreen" class="btn btn-outline-success btn-sm backToTop" onclick="topFunction()">Go To Top</button>
             
             <div class="btn-group float-right" role="group" aria-label="Button Group">
+                <button id="showOnlyFailures" class="btn btn-outline-success btn-sm float-right">Show Only Failures</button>
```

Comment:
If you also add this, it will hide that button from the UI if there are no failed iterations.

```
{{#if summary.failures.length}}
<button id="showOnlyFailures" class="btn btn-outline-success btn-sm float-right">Show Only Failures</button>
{{/if}}
```